test/fw/bin/pbs_benchpress

```diff
@@ -340,6 +348,42 @@ if __name__ == '__main__':
             'Testsuites names require (see -t) along with -i option!')
         sys.exit(1)
 
+    try:
+        tc_failure_threshold = int(tc_failure_threshold)
+        if tc_failure_threshold < 0:
+            raise ValueError
+    except:
```

Comment:
@arungrover If I understand this correctly then you mean to say that remove try and except block while we are checking whether given threshold count is integer or not (i.e. int(<count>))?
If Yes, then we can not remove try and except as while doing typecase (i.e. int()), if given value can not be typecast-ed then int() will throw ValueError exception. So to catch that I used try and except, and print error message as per DD in except block then exit().
Hope you got my point.